packages/stimulus-library/cypress.config.ts

```diff
@@ -27,6 +27,6 @@ export default defineConfig({
     setupNodeEvents(on, config) {
       return require("./cypress/plugins/index.js")(on, config);
     },
-    baseUrl: "http://localhost:3000/",
+    baseUrl: "http://localhost:3030/",
```

Comment:
When running locally it looks like vite is configured to run on 3030 and that's where cypress should be looking so updated this.